### DISCUSSION TITLE
Update image on hub.docker.com?

### DISCUSSION THREAD

**Discussion by klara31** [abraunegg/onedrive#1736] [Q&A] [answered]
Recently, the image on git has been updated. Just wondering: when will this new image be pushed to hub.docker.com?

**Comment by abraunegg** [comment]
@klara31 
Docker has changed their account structure, so that free accounts cannot integrate to GitHub as per what was occurring prior. You now can only build via a paid account

An application to Docker re being open source has been done, and PR #1733 also provides an alternate method.

**Comment by abraunegg** [comment, marked as answer]
@klara31 
With the release of v2.4.18 - Docker containers are being automatically built and pushed to Docker Hub. Please check [docker.md](https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md) for updated Docker Tag details.
